### PR TITLE
Commands & MultiTask improvements

### PR DESCRIPTION
**Pull request by Aurelien30000**
- Add ``/land clearinactive`` to clear inactive players' lands.
- Refactor former _clearworld_ command, this name was not consistent since ``/landlord clear`` can clear players, worlds or all lands.
- Bump versions and config versions due to breaking changes and additions!

Improvements:
- Don't run some parts of MultiTasks code if called by server shutdown. _Tasks can't be ran._
- Fix duplicate Debug command :thinking:
- Fix pl translation.
- Fix some commands not correctly checking args range before getting a value.
- Make ``noPermissionsCmd`` translateable.
- Improve MultiTask management, add some methods to ensure ability to process or clear.
- Add some logging during plugin shutdown.
- Remove useless chunk getters

### CODE REVIEW

File: LandLord-api/src/main/java/biz/princeps/landlord/api/IMultiTaskManager.java [hunk 1]
```diff
@@ -15,10 +15,17 @@
...
      */
-    void enqueueTask(IMultiTask multiTask);
+    void enqueueTask(IMultiTask<?> multiTask);
+
+    /**
+     * Clear all remaining queued tasks.
+     *
+     * @return the number of remaining tasks
+     */
+    int clear();
```
[the remark this answers is not in the document]
Comment (SirYwell) [reply LandLord-api/src/main/java/biz/princeps/landlord/api/IMultiTaskManager.java:29]: Adding a non-default method to an interface will break compilation of any implementation that does not override it. If the interface is not meant to be implemented by others, it should be somehow marked as internal. Common patterns for that are annotations (Jetbrains has `@ApiStatus.Internal` for example), package names containing `internal` or just a javadoc comment on the class/interface.

We might want to clean that stuff up with the next major version, but just for consistency it's probably better to make it `default` for now

File: LandLord-api/src/main/java/biz/princeps/landlord/api/IMultiTask.java [hunk 2]
```diff
@@ -11,9 +11,27 @@
...
+     * @return if value has been successfully processed
+     */
+    boolean process(T value);
+
+    /**
+     * Check if operations should be processed.
+     *
+     * @return if operations can still be processed
+     */
+    boolean canProcess();
```
[the remark this answers is not in the document]
Comment (SirYwell) [reply LandLord-api/src/main/java/biz/princeps/landlord/api/IMultiTask.java]: Would it probably better to call it something like `canContinueProcessing()` or something along the lines?

[the remark this answers is not in the document]
Comment (Aurelien30000) [reply LandLord-api/src/main/java/biz/princeps/landlord/api/IMultiTask.java]: Would make more sense indeed ;)

File: LandLord-api/src/main/java/biz/princeps/landlord/api/IMultiTaskManager.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Aurelien30000) [reply LandLord-api/src/main/java/biz/princeps/landlord/api/IMultiTaskManager.java:29]: Thanks for explanations!

**Review by SirYwell — APPROVED** [review]

**Pull request merged**